src/App.jsx

```diff
@@ -33,29 +37,41 @@ function App() {
   const onStart = () => {
     setStart(false);
     setReset(false);
+    setShowResult(false);
   };
 
   const onReset = () => {
     setReset(true);
     setStep(0);
     setCorrectAnswers(0);
     setStart(true);
+    setShowResult(false);
   };
 
   useEffect(() => {
     const fetchData = async () => {
       try {
-        const res = await axios.get(API_URL);
+        const userToken = await axios.get(TOKEN);
```

Comment:
The user token should be received only once when the user starts playing. All future requests should reuse the same token until it expires.

Reply:
I'm not sure that I wrote it correctly